Reports/progress_report.md

```diff
@@ -0,0 +1,296 @@
+- Codebase pre-release https://github.com/alan-turing-institute/data-safe-haven/releases/tag/v5.0.0-rc.1
+- Next version milestone https://github.com/alan-turing-institute/data-safe-haven/milestone/20
+
+## Information governance & standards
+Infrastructure adhering to the latest agreed upon standard
+Identifying, co-creating and supporting a TRE standard used across TRE infrastructures
+
+### Co-create a TRE standard (SATRE)
+https://github.com/alan-turing-institute/data-safe-haven-team/issues/23
+
+Contributes to:
+- Identifying, co-creating and supporting a TRE standard used across TRE infrastructures
+
+#### Goal
+Develop the SATRE specification that UK TREs can evaluate themselves against
+
+##### Definition of done
+SATRE specification published
+
+#### Progress
+[SATRE specification](https://satre-specification.readthedocs.io/en/stable/) published, and available for contribution and reproducibility in its [open repository](https://github.com/sa-tre/satre-specification).
+
+Currently Turing and HIC have self-evaluated against it and evaluations are available openly, several conversations ongoing about other institutions doing the same and making them available.
+
+- Outputs:
+  - SATRE specification V1 and [associated technical paper](https://zenodo.org/records/10053383)
+  - Internal DARE report, link will be added and published openly
+  - [User report](https://zenodo.org/records/10066800): characterises users, with a wider notion than we started of what users are. Lays the fundation for futher usability work, going beyong technicla features and into training and documentation
+  - UK TRE Community - SATRE WG: the ongoing work and evolution of the specification is now a working group within the Uk TRE Community
+
+### Documentation management
+https://github.com/alan-turing-institute/data-safe-haven-team/issues/32
+
+Contributes to:
+- Identifying, co-creating and supporting a TRE standard used across TRE infrastructures
+- Creating resources for all stakeholders (inc. Citizens) to engage in the TRE conversation
+- Identifying and documenting everything that can be openly documented
+
+#### Goal
+Comprehensive and clear documentation for the DSH, SATRE & TRESA will ensure open, reproducible outputs from this project.
+- Ensure all relevant information is captured in documentation
+- Test accessibility and discoverability of docs with relevant groups
+- Iterate documentation in line with wider project work (e.g. TRESA processes, DSH updates)
+- Determine what can/can't be documented (e.g. from an IG perspective).
+
+#### Definition of Done
+When funding ends for the project and we have openly documented everything that we feel we can
+
+#### Progress
+Documentation management have not been a story actively worked in, yet some processes have been updated and documented and needs identified.
```

Comment:
Definitly, we are working on that. We want to evaluate the stories we have under REG-TRESA in the roadmap with Helen, and adapt for her and myself to update.
For this report I reported for TRESA a whole because in practice we did not keep to those stories as they are formulated